unity/Assets/Scripts/ArtGallery/Util/NaiveLighthouseToLighthouseVisibility.cs

```diff
@@ -0,0 +1,279 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using Util.Algorithms.Polygon;
+using Util.Geometry;
+using Util.Geometry.Polygon;
+
+namespace ArtGallery
+{
+    public static class NaiveLighthouseToLighthouseVisibility
+    {
+        public static List<Vector2> VisibleVertices(
```

Comment:
Missing documentation, what do the polygon and vertex represent?

Reply:
Those were for testing. they are needed with the advanced visibility. I removed them in this branch along with their tests. Apologies